packages/mobile-top-nav/src/assets/img/hamburger.js

```diff
@@ -0,0 +1,31 @@
+import { html, css } from 'lit-element';
+import Icon from './icon';
+
+class HamBurger extends Icon {
+  static get styles() {
+    return css`
+      path {
+        stroke: var(--baseColor);
+      }
+      .active path {
+        stroke: var(--activeColor);
+      }
+    `;
+  }
+
+  render() {
+    return html`
+      <svg class="${this.active ? 'active' : ''}" height="40" viewBox="0 0 40 40" width="40" xmlns="http://www.w3.org/2000/svg" aria-labelledby="hamburgerTitleID hamburgerDescID">
+        <title id="hamburgerTitleID">Hamburger icon</title>
```

Comment:
heheheh 🍔  - we should probably name this main menu or something

Reply:
It was my understanding that the title was used to describe the image and the desc was used to provide an alternate definition of what the image represents.

Reply:
that's fine.